sdks/java/core/pom.xml

```diff
@@ -449,6 +455,12 @@
     </dependency>
 
     <dependency>
+      <groupId>net.bytebuddy</groupId>
+      <artifactId>byte-buddy</artifactId>
+      <version>1.4.3</version>
```

Comment:
I think this is a candidate for versioning in the `<dependencyManagement>` section.